package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sapphire-slash-commands",
   "version": "1.0.0",
   "description": "",
-  "main": "./src/index.js",
+  "main": "./javascript/index.js",
```

Comment:
I would create 2 different `package.json`'s, one for typescript and one for javascript.